src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int MAX_NUMBER = 6;
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        return Color.values()[random.nextInt(MAX_NUMBER)].name();
+    }

```

Comment:
in this impl, when you add a new color, you will need to remember to come here and change this value or one color will never be generated. instead of 6 depend on length of color values